### PR TITLE
Telemetry skeleton implementation

### PR DESCRIPTION
**Pull request by walinga**

### CODE REVIEW

File: d2l-rubric-criteria-group.html [hunk 1]
```diff
@@ -196,6 +202,11 @@
 				if (!entity) {
 					return;
 				}
+				// The first time that the entity loads, we send out an Open event
```
Comment (alogan519) [inline d2l-rubric-criteria-group.html]: looks like it's just a "TelemetryEvent"

File: telemetry-behavior.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
+	window.D2L.PolymerBehaviors.Rubric = window.D2L.PolymerBehaviors.Rubric || {};
+
+	/**
+	 * Behavior for sending telemetry messages to the telemetry service
+	 * @polymerBehavior
+	 */
+	D2L.PolymerBehaviors.Rubric.TelemetryBehaviorImpl = {
+
+		_logEvent: function(entity, isMobile, eventType) {
```
Comment (djwester) [inline telemetry-behavior.html]: Do we have any specs on what/when we want to log things?

File: d2l-rubric-criteria-group.html (hunk 1 above)
Comment (djwester) [inline d2l-rubric-criteria-group.html]: Should we have a way to turn this on/off? - Maybe LD flag?

File: telemetry-behavior.html (hunk 2 above)
Comment (walinga) [reply telemetry-behavior.html]: The [feature](https://rally1.rallydev.com/#/57732444928d/detail/portfolioitem/feature/210511399256) asks for a number of things that we should log. My plan is to have `isMobile`, `rubricMode`, and `originTool` on the default telemetry event, along with standard event data (tenant ID, browser, etc.). Then on the performance event, we can add in metrics for rubric rendering.

Not sure if this is the right spot that we want to be sending the event, but that's flexible so we can easily change it if we need to.

File: d2l-rubric-criteria-group.html (hunk 1 above)
Comment (walinga) [reply d2l-rubric-criteria-group.html]: The event `Action` is "Open". We might add different event types in the future (eg. a "Close" event), so I wanted to specify what kind of event we are sending

Comment (walinga) [reply d2l-rubric-criteria-group.html]: LD flag coming soon in a separate PR

**Pull request merged**